src/components/VContentSwitcherOld/VSearchTypeButtonOld.vue

```diff
@@ -5,7 +5,8 @@
       sizeClasses,
       {
         'max-w-[10rem] sm:max-w-[20rem] md:max-w-[16rem]': isHeaderScrolled,
```

Comment:
This change is not related to the desktop searchbar, but it fixes the homepage: when the desktop search type button is focused, it should be white and should not have a border (it has a pink ring instead).